src/queryBuilder/Rule.js

```diff
@@ -35,7 +36,9 @@ let Rule = ({
           }}
           {...F.domLens.hover(state.lens.ruleHover)}
         >
-          <FilterContents {...{ node, tree, Button, Modal, Picker }} />
+          <FilterContents
+            {...{ node, tree, Button, Modal, Picker, ...props }}
```

Comment:
Rule doesn't use any of these three components itself, so there's no need to pull them out and pass them explicitly now that we're passing down a `...props` mystery box anyway.

I still don't love this pattern, but we should at least be consistent about using it